README.md

```diff
@@ -1,32 +1,72 @@
 This is an experiment to create a Docker container that approximates
-the runtime environment of a cloud.gov/CloudFoundry droplet
-using a Python buildpack on the [cflinuxfs2][] stack.
+the runtime environment of a cloud.gov/CloudFoundry droplet on the
+[cflinuxfs2][] stack.
 
 The Dockerfile is heavily inspired by [sclevine/cflocal][].
 
 ## Quick start
+To build an image for a particular buildpack, you provide  the language
+as a argument to the build script. For example, to build an image for `python`:
 
+```shell
+./build.sh python
 ```
-docker-compose build
-docker-compose run app python
+
+This will build a base image that can be used by your own application
+to replicate the build lifecycle when pushing an application.
+
+### Using in your application
+Once you have
+built this image, you can then add a Dockerfile to your own application
+that looks like:
+
+```Dockerfile
+FROM cloud-gov/python
```

Comment:
It might be worth explaining why an almost completely empty `Dockerfile` is needed, rather than just doing `image: cloud-gov/python` in the `docker-compose.yml`?  I *think* it is because an actual `Dockerfile` is required for the `ONBUILD` instructions to trigger, but I could be wrong.

OTOH I am a Docker noob so maybe this will be obvious to more folks :)